package-lock.json

```diff
@@ -0,0 +1,2642 @@
+{
+  "name": "generator-az-terra-module",
+  "version": "0.2.1",
```

Comment:
Please update your pacakge.json to `0.3.0`, add git tag, and do publish after merge.

Reply:
Will do the publish later to include another change related to LICENSE wizard.